be/src/olap/rowset/segment_v2/column_reader.cpp

```diff
@@ -466,6 +486,42 @@ Status ArrayFileColumnIterator::next_batch(size_t* n, ColumnBlockView* dst, bool
     return Status::OK();
 }
 
+Status ArrayFileColumnIterator::_seek_by_offsets(ordinal_t ord) {
+    // using offsets info
+    ordinal_t offset = 0;
+    RETURN_IF_ERROR(_peek_one_offset(&offset));
+    RETURN_IF_ERROR(_item_iterator->seek_to_ordinal(offset));
+    return Status::OK();
+}
+
+Status ArrayFileColumnIterator::seek_to_ordinal(ordinal_t ord) {
+    RETURN_IF_ERROR(_offset_iterator->seek_to_ordinal(ord));
+    if (_array_reader->is_nullable()) {
+        RETURN_IF_ERROR(_null_iterator->seek_to_ordinal(ord));
+    }
+    return _seek_by_offsets(ord);
+}
+
+Status ArrayFileColumnIterator::_caculate_offsets(ssize_t start,
+                                                  vectorized::MutableColumnPtr& offsets,
+                                                  size_t* num_items) {
```

Comment:
```suggestion
Status ArrayFileColumnIterator::_calculate_offsets(vectorized::ColumnArray::ColumnOffsets& offsets, ssize_t start)
```

1. Typo: `_caculate_offsets` -> `_calculate_offsets`
2. For function arguments, I think it is better to put the mutable arguments before the immutable ones.

Reply:
2. But I saw many doris code which put immutable before mutable arguments

Reply:
done